src/registries/srt_registry.ts

```diff
@@ -24,6 +24,11 @@ export class SpecificRangeTransformRegistry extends Registry<
     return this;
   }
 
+  replace<C extends CoreCommand>(cmdType: C["type"], fn: CommandAdaptRangeFunction<C>): this {
+    super.replace(cmdType, fn);
+    return this;
+  }
```

Comment:
I don't think we need/want to allow replace for this registry

Reply:
I feel like between having it work (and be explicit) or having it throw an error, it might as well be there :shrug:.
If you feel strongly about it, I can make it err instead.

Reply:
I don't feel that strongly about it 🤷 

But actually do we even need to override `replace` and `add` in this registry ? All we do is call `super`

Reply:
The override are only present for typing purposes (as it already was)